### PR TITLE
Fix linux

### PR DESCRIPTION
**Pull request by nop33**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -104,17 +104,29 @@ const App = () => {
...
+    network.name,
+    posthog.__loaded,
+    posthog.people,
+    settings.devTools,
+    settings.fiatCurrency,
+    settings.language,
+    settings.passwordRequirement,
+    settings.theme,
+    settings.walletLockTimeInMinutes,
+    wallets.length
```
Comment (mvaivre) [inline src/App.tsx:128]: Hahaha when the deps array is longer that the actual useEffect 😆

File: src/contexts/analytics.tsx [hunk 2]
```diff
@@ -23,7 +23,9 @@ import AnalyticsStorage from '@/storage/analytics/analyticsPersistentStorage'
 import SettingsStorage from '@/storage/settings/settingsPersistentStorage'
 import { GeneralSettings } from '@/types/settings'
 
-const { VITE_PUBLIC_POSTHOG_KEY, VITE_PUBLIC_POSTHOG_HOST, VITE_VERSION } = import.meta.env
+const VITE_PUBLIC_POSTHOG_KEY = 'phc_FLKGQDmMQSdSb3qjaTwHWwm9plmz7couyVJFG9GOMr7'
+const VITE_PUBLIC_POSTHOG_HOST = 'https://eu.posthog.com'
+const VITE_VERSION = import.meta.env
```
Comment (mvaivre) [inline src/contexts/analytics.tsx:28]: 1) If we don't use env variables, let's simplify the names by removing the VITE_ prefix?
2) Line 28 looks fishy

Comment (nop33) [reply src/contexts/analytics.tsx:28]: you're totally right

Comment (nop33) [reply src/contexts/analytics.tsx]: fixed

**Review by mvaivre — APPROVED** [review]
LGTM, well done

**Pull request merged**